### DISCUSSION TITLE
How can I insert the element in specific location?

### DISCUSSION THREAD

**Discussion by poppintiger** [zauberzeug/nicegui#1364] [Q&A]
### Description

if I want put a button in the third column, how can I do ?
![image](https://github.com/zauberzeug/nicegui/assets/26161307/7cfa9b7f-c64f-4429-bcaa-49d7f876368a)

I try this,but it did not work, so,how can I deal with it ?
```py
with ui.row():
    ui.label(' ')
    ui.label(' ')
    ui.button('chat', on_click=lambda: ui.open('documentation#open')).style('size:3px')
```

**Comment by rodja** [comment]
The empty strings are not very wide, so the button will look like its completely to the right. You should set width attributes on the labels.

**Comment by falkoschindler** [comment]
@poppintiger What is not working? What do you expect to happen?

**Comment by falkoschindler** [comment]
By the way, @poppintiger: Please use [Q&A discussions](https://github.com/zauberzeug/nicegui/discussions/categories/q-a), [Discord](https://discord.com/invite/TEpFeAaF4f) or [StackOverflow](https://stackoverflow.com/questions/tagged/nicegui) for asking questions. Only open an issue if there's something wrong with NiceGUI.
